site/oauth2-examples.md

```diff
@@ -161,9 +172,7 @@ with a single click. The web portal retrieves a token before taking the user to
       1. rabbit_admin from a browser                                   3. validate token        
 </pre>
 
-How it works, firstly, the `rabbit_admin` user navigates to the web portal and clicks on the hyperlink associated with a RabbitMQ
-cluster. Next, the web portal obtains a token and redirects the user to RabbitMQ `/login` endpoint with the token within the HTTP form field `access_token`. Finally,
-RabbitMQ validates the token in the http request and if it is valid, it redirects the user to the overview page.
+If the access token is valid, RabbitMQ redirects the user to the overview page.
```

Comment:
"is "Overview page" the actual name of a page on the "RabbitMQ Management UI" if so bold it and name it exactly as it is on the UI: **Overview** page